### PR TITLE
Update README for peewee-moves

### PR DESCRIPTION
**Pull request by pylipp**
Not going too much into detail to avoid duplicating the peewee-moves documentation

### CODE REVIEW

File: flask/README.md [hunk 1]
```diff
@@ -200,4 +200,13 @@ A sample query you can try if it works is:
 
 ## Authentication and Authorization
 
-## Database Migrations
+## Database Schema Migrations
+
+Occasionally it might be required to update the database schema. To this end we use the [peewee-moves](https://github.com/timster/peewee-moves) tool.
```
Comment (aerinsol) [inline flask/README.md:205]: Why not use the official repo? (Coleifer)

**Review by aerinsol — CHANGES_REQUESTED** [review]
Maybe make a quick comment about rollbacks and rollforwards in versioning.

I'd also like to see aside from your comment about needing to date the schema, if you're altering the schema it's good practice to date /version it.

**Comment by aerinsol** [comment]
No issue with not replicating the peewee documentation, just being clear about how the structure ties together. 

Also would be nice to tie in with a general commentary on peewee as well and how it's used here as an ORM that sits on top of the DB.

File: flask/README.md (hunk 1 above)
Comment (pylipp) [reply flask/README.md:205]: coleifer is the maintainer of peewee itself. I didnt mention it since the section is about the peewee-moves tool.

**Review by aerinsol — APPROVED** [review]

**Pull request merged**